fuse_loss/fuse_plugins.xml

```diff
@@ -1,50 +1,66 @@
 <library path="lib/libfuse_loss">
-  <class type="fuse_loss::TrivialLoss" base_class_type="fuse_core::Loss">
+  <!--
+  Loss functions are M-estimators that take the squared norm of a residual block 'r' and try to reduce the effect of
+  outliers by replacing the squared residuals 's = r^2' used in the standard Least-Squares problem by another function
+  of the squared residuals `rho(s)`.
+
+  For this reason loss functions are also known as robustifiers.
+
+  In the notation used here, `rho(s)` takes the squared residuals `s` as an input, not the residuals `r`, as in Ceres
+  solver.
+  -->
+  <class type="fuse_loss::ArctanLoss" base_class_type="fuse_core::Loss">
```

Comment:
Simply add the `fuse_loss::ScaledLoss` plugin and alphabetized everything, apart from the comment above.